example/android/app/build.gradle

```diff
@@ -116,6 +116,7 @@ dependencies {
     // The version of react-native is set by the React Native Gradle Plugin
     implementation("com.facebook.react:react-android")
     implementation("com.facebook.react:flipper-integration")
+    implementation 'com.google.firebase:firebase-messaging-ktx:24.0.0'
```

Comment:
huh is this not a transitive dependency?

Reply:
Do we have this instruction in our readme or is it just something we are using for this test app?

Reply:
Nope, I do wonder if it should be.

Reply:
Should you add a comment about why this is needed?

Reply:
Good call bc i already forget why I had to add it

Reply:
Unresolved: didn't actually add a comment here!